core/src/main/java/nu/yona/server/analysis/service/DayActivityDTO.java

```diff
@@ -27,11 +41,14 @@
 	private final int totalMinutesBeyondGoal;
 	private final Set<MessageDTO> messages;
 
-	private DayActivityDTO(UUID goalID, ZonedDateTime startTime, boolean shouldSerializeDate, List<Integer> spread,
-			Optional<Integer> totalActivityDurationMinutes, boolean goalAccomplished, int totalMinutesBeyondGoal,
+	private final UUID activityCategoryID;
```

Comment:
It's here to be able to convert from``DayActivityDTO`` to ``DayActivityWithBuddiesDTO``. As there is no ``WeekActivityWithBuddiesDTO``, this is not needed for its week counterpart.